weekly-updates/weekly-update.2015-07-24.md

```diff
@@ -0,0 +1,14 @@
+# io.js and Node.js News
+This week we didn't have io.js releases. complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md).
+
+### Community Updates
+
+* Rod Vagg posted [an article about Node.js LTS](https://medium.com/@nodesource/essential-steps-long-term-support-for-node-js-8ecf7514dbd) on nodesource blog.
+* Daniel Kahn have written a blog post about [NODE_ENV in Express](http://apmblog.dynatrace.com/2015/07/22/the-drastic-effects-of-omitting-node_env-in-your-express-js-applications/).
```

Comment:
Please change into Daniel Khan (h after K)

Thank you


> Am 31.07.2015 um 07:14 schrieb Rich Trott <notifications@github.com>:
> 
> In weekly-updates/weekly-update.2015-07-24.md:
> 
> > @@ -0,0 +1,14 @@
> > +# io.js and Node.js News
> > +This week we didn't have io.js releases. complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md).
> > +
> > +### Community Updates
> > +
> > +* Rod Vagg posted [an article about Node.js LTS](https://medium.com/@nodesource/essential-steps-long-term-support-for-node-js-8ecf7514dbd) on nodesource blog.
> > +* Daniel Kahn have written a blog post about [NODE_ENV in Express](http://apmblog.dynatrace.com/2015/07/22/the-drastic-effects-of-omitting-node_env-in-your-express-js-applications/).
> Nit: Daniel Kahn has rather than Daniel Kahn have.
> 
> —
> Reply to this email directly or view it on GitHub.
>